### PR TITLE
Color augmentation added for lineAreaGraph, dayjs advance format and tooltip handler fix

### PR DESCRIPTION
**Pull request by whitetiger1399**
1. baseColor props if not defined then the colors will be taken from `palette.graph` and the corresponding graph series type.
2. advance plugin for dayjs added  
3. Tool tip handler error fix
Screenshot from 2021-06-01 12-00-00

Signed-off-by: Ritik Srivastava <ritik@chaosnative.com>

### CODE REVIEW

**Review by arkajyotiMukherjee — APPROVED** [review]

**Review by vanshBhatia-A4k9 — APPROVED** [review]

File: src/lab/Graphs/LineAreaGraph/LineAreaGraph.stories.mdx [hunk 1]
```diff
@@ -28,6 +28,7 @@ in form of lines, area-closed under the graph and events.
       yLabelOffset: 35,
       marginLeftEventTable: 10,
       widthPercentageEventTable: 40,
+      xAxistimeFormat: "hh:mm:ss Do MMM",
```
Comment (ishangupta-ds) [inline src/lab/Graphs/LineAreaGraph/LineAreaGraph.stories.mdx:31]: Do or DD? @whitetiger1399

Comment (whitetiger1399) [reply src/lab/Graphs/LineAreaGraph/LineAreaGraph.stories.mdx:31]: `Do` is for day of the month with ordinal i.e. `1st 2nd ...31st` and `DD` is for day of the month in 2-digits `01-31`

Comment (vanshBhatia-A4k9) [reply src/lab/Graphs/LineAreaGraph/LineAreaGraph.stories.mdx:31]: Do as seen in docs -> https://day.js.org/docs/en/plugin/advanced-format

**Review by ishangupta-ds — APPROVED** [review]

File: src/lab/Graphs/LineAreaGraph/LineAreaGraph.stories.mdx (hunk 1 above)
Comment (ishangupta-ds) [reply src/lab/Graphs/LineAreaGraph/LineAreaGraph.stories.mdx:31]: Looks good then.

**Pull request merged**